src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -89,6 +89,7 @@ import PlotLegend, {
 } from '@veupathdb/components/lib/components/plotControls/PlotLegend';
 import { isFaceted } from '@veupathdb/components/lib/types/guards';
 import FacetedPlot from '@veupathdb/components/lib/plots/FacetedPlot';
+import { config } from 'process';
```

Comment:
Oops! I'm guessing you don't need this import?

Reply:
@jtlong3rd Thank you for your review and suggestions! 👍 IDK why it was entered but perhaps due to a sort of VS Code's function which is often not that smart 😄